homeworks/maksym.selenkov_MaksymSelenkov/5-rxjs/index.html

```diff
@@ -0,0 +1,142 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>5-rxjs</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container" data-container>
+            <header class="header" data-header>
+                <span class="header__logo">Logo</span>
```

Comment:
In general, when you click on a logo, you expect to be redirected to the main page. For this purpose, you need to mark your logo as a link, not a span. I consider using the `a` tag here.